PIL/Image.py

```diff
@@ -2223,6 +2317,7 @@ def _check_size(size):
 
 
 def new(mode, size, color=0):
+    # type: (Mode, Size, Optional[Union[int, float, Tuple, Text]]) -> Image
```

Comment:
* Colorspec is preferable to eg. simply `Color`?
* Are all 'color' uses going to be of that type, eg. for getpixel/putpixel and no doubt others?
* I'm not sure if we gain anything with having `Union[int, float]` as part of that, cf `float`?

Either way, something like:
`Union[int, float, Tuple[int], Tuple[int, int], Tuple[int, int, int], Tuple[int, int, int, int], Text]` 
or (simplified, if the Tuple values are still `int`s?):
`Union[float, Tuple[int], Tuple[int, int], Tuple[int, int, int], Tuple[int, int, int, int], Text]`?